### ISSUE TITLE
Firebase Performance Monitoring is not working with Firebase Analytics

### ISSUE THREAD

**Issue by Mushtaq123** [xamarin/GooglePlayServicesComponents#290]
### Xamarin.Android Version (eg: 6.0):
7.1

### Operating System & Version (eg: Mac OSX 10.11):
Windows 10

### Google Play Services Version (eg: 8.4.0):
**60.1142.1** and the latest version **(71.1740.0)**

### Describe your Issue:
**For Latest Version:**

- Firebase Performance Monitoring is not working with Firebase Analytics.

We have already referred the below link and will not be worked for resolving the issue [https://github.com/xamarin/GooglePlayServicesComponents/issues/249](https://github.com/xamarin/GooglePlayServicesComponents/issues/249)

**For 60.1142.1**

- Firebase Performance Monitoring is working partially with Firebase Analytics.

- [X] Device Monitoring is working
- [ ] Network Monitoring is not working

We have already referred the below link and will not be worked for resolving the issue [https://github.com/xamarin/GooglePlayServicesComponents/issues/172](https://github.com/xamarin/GooglePlayServicesComponents/issues/172)

### Steps to Reproduce (with link to sample solution if possible):
1. Create a new project
2. Include Firebase Analytics and Performance Monitoring dependencies
3. Build and deploy the app in your Android Device.
4. Then Open

### Include any relevant Exception Stack traces, build logs, ADB logs:
<img src="https://raw.githubusercontent.com/Mushtaq123/ErrorBox/master/firebase_issue.jpg"/>

Please help us to resolve the same.

**Comment by moljac** [comment]
@Mushtaq123 
Did you add Xamarin.Android.ManifestMerger package?

**Comment by moljac** [comment]
Nuget url (FYI)

https://www.nuget.org/packages/Xamarin.Android.ManifestMerger/1.0.0-preview03

**Comment by Mushtaq123** [comment]
Yes, we have tried Android manifest merger NuGet and still, the issue replicates.

Please share if you have any samples to use Firebase Analytics and Firebase performance monitoring together

**Comment by rgustavsson** [comment]
@moljac how long is the Xamarin.Android.ManifestMerger package intended to be in preview?

**Comment by moljac** [comment]
@rgustavsson I don't know. Not a decision merger (maker).

**Comment by moljac** [comment]
@Mushtaq123 @rgustavsson 

No nuget package needed. We have moved this feature into the core of the Android build system.

https://docs.microsoft.com/en-us/xamarin/android/release-notes/10/10.2#improved-android-manifest-merging

All you need is to add:

```
<PropertyGroup>
  <AndroidManifestMerger>manifestmerger.jar</AndroidManifestMerger>
</PropertyGroup>
```
to your project[s] that reference GPS-FB packages.

Please report if this helps @Mushtaq123

**Comment by Mushtaq123** [comment]
We have tried the above-mentioned point and the issue highlighted issue is not occurring. but still, the network monitoring is not working

**Comment by Mushtaq123** [comment]
Dear Team,

Can we get an update on this issue? Also, suggest that the issue may work based on AndroidX or the latest version of the play services.

**Comment by moljac** [comment]
@Mushtaq123 

Providing a minimal up to date repro sample would help a lot. Could you, please? Thanks

**Comment by 000bravvo** [comment]
Hi, I'm getting the same issue as above. I tried all the solutions given but nothing worked. Can someone please update a solution or any workaround available? Thanks
